test/integration/Deployment.t.sol

```diff
@@ -427,4 +421,33 @@ contract DeploymentTest is DssTest {
         vm.prank(facilitator1); ConduitMover(ilkInst.conduitMover).setConfig(conduit1, conduit2, USDC, 1, 1 hours, 3_000 * 10**6);
         vm.prank(conduitMoverKeeper1); ConduitMover(ilkInst.conduitMover).move(conduit1, conduit2, USDC);
     }
+
+    function testEndCage() public {
```

Comment:
Consider adding a comment that this test does not imply that ES is supported, but it just makes sure that the ink amount selection adheres with the End logic.

Reply:
Done.